packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -64,11 +64,18 @@ const propTypes = {
    * Object containing intl APIs
    */
   intl: PropTypes.shape({ formatMessage: PropTypes.func }),
+
+  /**
+   * @private
+   * Allows to reset cache and elements will be rendered face-up for width calculations.
+   */
+  allowResetCache: PropTypes.bool,
```

Comment:
Why is this prop required if the default value is true?

Reply:
In  mpage application side, they don't need this reset cache method to execute. so kept this as a internal prop. where they can send as false.